ci/openstack/install.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source ci/openstack/vars.sh
+if [ "${RUN_OPENSTACK_CI:-}" != "true" ]; then
+    echo RUN_OPENSTACK_CI is set to false, skipping the openstack end to end test.
+    exit
+fi
+
+if [ "${CI_OVER_CAPACITY:-}" == "true" ]; then
+    echo the CI is over capacity, skipping the end-end test.
+    exit 1
+fi
+
+git clone https://github.com/openshift/openshift-ansible ../openshift-ansible
+cd ../openshift-ansible
+git checkout openshift-ansible-3.6.153-1
```

Comment:
Yeah, I was thinking of an env var for travis to get the needed version , or use the latest by default